src/com/apeshko/javacore/task_01/model/Airplane.java

```diff
@@ -0,0 +1,108 @@
+package com.apeshko.javacore.task_01.model;
+
+public class Airplane extends FlyingMachine {
+    public static enum Type {
+        BOEING,
+        AIRBUS,
+    }
+
+    private Airplane.Type type;
+
+    public Type getType() {
+        return type;
+    }
+
+    public void setType(Type type) {
+        this.type = type;
+    }
+
+    public static class Builder {
+        private Airplane airplane;
+
+        public Builder() {
+            airplane = new Airplane();
+        }
+
+        public Builder withName(String name) {
+            airplane.setName(name);
+
+            return this;
+        }
+
+        public Builder withWeight(double weight) {
+            airplane.setWeight(weight);
+
+            return this;
+        }
+
+        ;
```

Comment:
formatting;)
if you use intellij, try select content => ctrl+alt+l to make it easier